### DISCUSSION TITLE
（已解决）群晖CLOUD SYNC同步“上传加密密钥失败”

### DISCUSSION THREAD

**Discussion by yefq1984** [AlistGo/alist#1068] [General]
ALIST挂了天翼云盘和阿里云盘，用CLOUD SYNC同步WEBDAV，提示“上传加密密钥失败”
![image](https://user-images.githubusercontent.com/105204314/167397597-394c25ab-5a45-43e2-ae41-ef8a88121935.png)

**Comment by xiaoshen** [comment]
同样问题，大佬解决了么

**Comment by lyg367090** [comment]
Cloud Sync在传送加密文件的时候，会在目标盘根目录新建@SynologyCloudSync文件夹来存储加密信息，如果你在alist里挂载多个网盘，那么对Cloud Sync来说，根目录就不可写，于是出错。可以尝试在alist里只挂载阿里云盘到/，或者在Sync里填写WebDAV服务器时写serverip:port/dav/xxx来解决。xxx是你在alist里挂载阿里云盘的目录名

**Reply by yefq1984** [reply]
可行，非常感谢大神。ALIST只挂一个阿里云盘不好用。
我用的是在CloudSync填写WebDAV服务器时加云盘目录解决，这样，如果有多个就添加多个任务就行。再次感谢

**Reply by yefq1984** [reply]
> Cloud Sync在传送加密文件的时候，会在目标盘根目录新建@SynologyCloudSync文件夹来存储加密信息，如果你在alist里挂载多个网盘，那么对Cloud Sync来说，根目录就不可写，于是出错。可以尝试在alist里只挂载阿里云盘到/，或者在Sync里填写WebDAV服务器时写serverip:port/dav/xxx来解决。xxx是你在alist里挂载阿里云盘的目录名

大神，再问个问题。我在用ALIST上传文件到天翼云盘，用的189PC那个端口，上传有些文档会出错，一般都是文本文件或者游戏存档。但是，也不全出错，上传照片什么的就正常。只要是会出错的文件，就算打包成压缩包上传还是一样出错。用原来那个189的WEB端口就正常（但是，这个上传同名文件会重命名，我需要上传覆盖）。出错的截图如下：
![image](https://user-images.githubusercontent.com/105204314/174751105-8da36c40-32d6-4e7e-9bf5-002f59555349.png)

**Reply by jony-hub** [reply]
> 一直以来，非常感谢大神。ALIST只挂一个阿里云不好用。 我用的是在CloudSyncWebDAV服务器时加云目录解决，这样如果有多个盘，就添加多个任务。再次感谢

我设置了根/dav/阿里云盘 就不可以添加其他云盘了，提示说已经有任务，只能在阿里云盘下面再添加子任务，不知道兄台如何解决的，求解，谢谢

**Reply by cybaijie** [reply]
> > 一直以来，非常感谢大神。ALIST只挂一个阿里云不好用。 我用的是在CloudSyncWebDAV服务器时加云目录解决，这样如果有多个盘，就添加多个任务。再次感谢
> 
> 我设置了根/dav/阿里云盘 就不可以添加其他云盘了，提示说已经有任务，只能在阿里云盘下面再添加子任务，不知道兄台如何解决的，求解，谢谢

找到解决办法了，先建一个本地存储，然后本地存储放到根目录上，再把其他网盘建在本地存储里就可以了

**Reply by AresSun112** [reply]
> > > 一直以来，非常感谢大神。ALIST只挂一个阿里云不好用。 我用的是在CloudSyncWebDAV服务器时加云目录解决，这样如果有多个盘，就添加多个任务。再次感谢
> > 
> > 
> > 我设置了根/dav/阿里云盘 就不可以添加其他云盘了，提示说已经有任务，只能在阿里云盘下面再添加子任务，不知道兄台如何解决的，求解，谢谢
> 
> 找到解决办法了，先建一个本地存储，然后本地存储放到根目录上，再把其他网盘建在本地存储里就可以了

你好，可以详细说一下操作步骤吗？

**Reply by N04LE7** [reply]
> > > > 一直以来，非常感谢大神。ALIST只挂一个阿里云不好用。我用的是在CloudSyncWebDAV服务器时加云目录解决，这样如果再有多个盘，就添加多个。
> > > 
> > > 
> > > 我设置了根/dav/阿里云盘就不能添加其他云盘了，提示说已经有任务，只能在阿里云盘下面再添加子任务，不知道，兄弟的视图如何查看
> > 
> > 
> > 找到解决办法了，先建一个本存储器，然后把本存储器放到根目录上，再把其他网盘建在本存储器里就可以了
> 
> 你好，可以详细说一下操作步骤吗？

把其中一个网盘的挂载路径只填写/，就可以解决了。

**Reply by SilenceOBrea1** [reply]
感谢，已解决

**Comment by MoConWu** [comment]
# 推荐一个对于Alist3简单的办法：
将本地的一个目录（例如：/volume1/home/alist-root）挂载到Alist上作为根目录，也就是【/】，本地的目录记得给Alist3权限，然后再创建就可以了。
# 注意：
但是这样的话，记得挂载在Alist3上的CloudSync的密码**必须都是一样的才行**，也就是**同一个密码**，因为在这个目录上共用了@SynologyCloudSync及加密文件。
